### PR TITLE
XEH - Use compileScript in CBA_fnc_compileFunction

### PR DESCRIPTION
**Pull request by PabstMirror**
ref https://community.bistudio.com/wiki/compileScript

### CODE REVIEW

File: addons/xeh/fnc_preInit.sqf [hunk 1]
```diff
@@ -26,7 +26,7 @@ SLX_XEH_DisableLogging = uiNamespace getVariable ["SLX_XEH_DisableLogging", fals
 XEH_LOG("PreInit started. v" + getText (configFile >> "CfgPatches" >> "cba_common" >> "versionStr"));
 
 SLX_XEH_STR = ""; // does nothing, never changes, backwards compatibility
-SLX_XEH_COMPILE = compileFinal "compile preprocessFileLineNumbers _this"; //backwards comps
+SLX_XEH_COMPILE = compileFinal "diag_log text format ['[CBA-XEH] old SLX_XEH_COMPILE macro used on %1', _this]; compile preprocessFileLineNumbers _this"; //backwards comps
```
Comment (commy2) [inline addons/xeh/fnc_preInit.sqf:29]: ?

Also, whitespace EOL

File: addons/xeh/fnc_preStart.sqf [hunk 2]
```diff
@@ -23,7 +23,7 @@ with uiNamespace do {
 
     XEH_LOG("PreStart started.");
 
-    SLX_XEH_COMPILE = compileFinal "compile preprocessFileLineNumbers _this"; //backwards comps
+    SLX_XEH_COMPILE = compileFinal "diag_log text format ['[CBA-XEH] old SLX_XEH_COMPILE macro used on %1', _this]; compile preprocessFileLineNumbers _this"; //backwards comps
```
Comment (commy2) [inline addons/xeh/fnc_preStart.sqf:26]: same

File: addons/xeh/fnc_preInit.sqf (hunk 1 above)
Comment (PabstMirror) [reply addons/xeh/fnc_preInit.sqf:29]:
```suggestion
SLX_XEH_COMPILE = compileFinal "diag_log text format ['[CBA-XEH] old SLX_XEH_COMPILE macro used on %1', _this]; compile preprocessFileLineNumbers _this"; //backwards comps
```

File: addons/xeh/fnc_preStart.sqf (hunk 2 above)
Comment (PabstMirror) [reply addons/xeh/fnc_preStart.sqf:26]:
```suggestion
    SLX_XEH_COMPILE = compileFinal "diag_log text format ['[CBA-XEH] old SLX_XEH_COMPILE macro used on %1', _this]; compile preprocessFileLineNumbers _this"; //backwards compat
```

**Review by commy2 — APPROVED** [review]

**Pull request merged**